src/System.IO.Compression.ZipFile/tests/ZipFileConvenienceMethods.cs

```diff
@@ -77,24 +77,37 @@ public static void ExtractToDirectoryNormal()
             TestExtract(zfile("noexplicitdir.zip"), zfolder("explicitdir"));
         }
 
-        private static void TestExtract(String zipFileName, String folderName)
+        private void TestExtract(string zipFileName, string folderName)
         {
-            String tempFolder = StreamHelpers.GetTmpPath(true);
+            string tempFolder = GetTmpDirPath(true);
             ZipFile.ExtractToDirectory(zipFileName, tempFolder);
             DirsEqual(tempFolder, folderName);
+
+            Assert.Throws<ArgumentNullException>(() => ZipFile.ExtractToDirectory(null, tempFolder));
         }
 
         #region "Extension Methods"
 
-        [Fact]
-        public static async Task CreateEntryFromFileTest()
+        [Theory]
+        [InlineData(true)]
+        [InlineData(false)]
```

Comment:
InlineData can be used any number of times.  xunit runs the test once for each InlineData, using the arguments to the attribute as the arguments to the test function.  So in this case, CreateEntryFromFileTest will be called twice, once with true and once with false.